doc/rbd/nvmeof-initiator-linux.rst

```diff
@@ -0,0 +1,83 @@
+==============================
+ NVMe/TCP Initiator for Linux
+==============================
+
+Prerequisites
+=============
+
+- Kernel 4.19 or later
+- RHEL 9.2 or later
+- Ubuntu 24.04 or later
+- SLES 15 SP3 or later
+
+Installation
+============
+
+1. Install the nvme-cli:
+
+   .. prompt:: bash #
+   
+      yum install nvme-cli
+
+2. Install the NVMe-oF module:
```

Comment:
Technically, this module is installed together with the kernel.  I'd suggest "add" or "load" here.

Reply:
Accepted.